src/pages/billing/models/BillingProject.ts

```diff
@@ -19,6 +19,7 @@ export interface BillingProject {
 export interface AzureBillingProject extends BillingProject {
   cloudPlatform: 'AZURE'
   managedAppCoordinates: AzureManagedAppCoordinates
+  landingZoneId: string
```

Comment:
I considered making this optional because if the landing zone creation fails to initiate I thought we may be left with an errored billing project with no landing zone ID. But looking through the current version of the Rawls code, it does appear we delete the billing project if initiation of the landing zone fails (and we get the ID back from that initiation call).